ws-client/Cargo.toml

```diff
@@ -21,12 +21,13 @@ jsonrpsee-types = { path = "../types", version = "0.3.0" }
 log = "0.4"
 serde = "1"
 serde_json = "1"
-soketto = "0.6"
+soketto = "0.7"
 pin-project = "1"
 thiserror = "1"
-url = "2"
+http = "0.2"
```

Comment:
Switched to the `http crate` instead for URI parsing because it supports both `absolute URI` and `relative resource URI`

Useful when getting the location header back...